src/tcgwars/logic/impl/gen4/Platinum.groovy

```diff
@@ -2660,9 +2663,13 @@ public enum Platinum implements LogicCardInfo {
       case POKE_TURN_118:
         return itemCard (this) {
           text "Return 1 of your Pokémon SP and all cards attached to it to your hand."
+          // TODO: Check for Pokémon SP in onPlay and playRequirement
           onPlay {
+            def pcs = my.bench.findAll { true }.select("Which Pokémon SP to return to hand?")
+            scoopUpPokemon(pcs, delegate)
           }
           playRequirement{
+            my.bench.any { true }
```

Comment:
Can be played on your only Pokémon in play. See: Super Scoop Up, review comment on Toxicroak.